Profile Folder/chrome/locale/ru-RU/dtd/gsettings.dtd

```diff
@@ -147,22 +147,22 @@
 <!ENTITY gsettings.themeToBehaviour             "Настройте, насколько сильно темы влияют на интерфейс...">
 <!ENTITY gsettings.themeToExperiments           "Персонализируйте дальше с помощью заброшенных опций Chromium...">
 
-<!ENTITY gsettings.peopleButton                 "[!!! Þèôƥℓè ßúƭƭôñ !!!]">
-<!ENTITY gsettings.peopleButtonStyle			"[!!! Þèôƥℓè ßúƭƭôñ §ƭ¥ℓè !!!]">
-<!ENTITY gsettings.avatarBesideTabs             "[!!! ÂƲáƭář βèƨïδè Táβƨ !!!]">
-<!ENTITY gsettings.titlebarButton               "[!!! Tïƭℓèβář ßúƭƭôñ !!!]">
+<!ENTITY gsettings.peopleButton                 "Кнопка Профили">
+<!ENTITY gsettings.peopleButtonStyle			"Выбор стиля кнопки профиля">
+<!ENTITY gsettings.avatarBesideTabs             "Аватар рядом с вкладками">
+<!ENTITY gsettings.titlebarButton               "Кнопка на панели вкладок">
```

Comment:
Is `gsettings.titlebarButton` mentioning "tab bar" or "titlebar"? It needs to be mentioning the titlebar.